tests/core/recipes/vasp_recipes/mocked/test_vasp_recipes.py

```diff
@@ -848,3 +849,14 @@ def test_mp_relax_flow_custom(tmp_path, patch_nonmetallic_taskdoc):
             job_params={"mp_metagga_relax_job": {"nsw": 0}},
         )
         assert output["relax2"]["parameters"]["nsw"] == 0
+
+
+def test_freq_job():
+    atoms = molecule("H2")
+    atoms.pbc = True
+    atoms.set_cell([3, 3, 3])
```

Comment:
Let's instead do

```python
atoms.center(vacuum=5, axis=2)
```

or something like that which is more akin to what one would do in reality.